tests/monotouch-test/System.Net.Http/NetworkResources.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using Foundation;
+
+namespace MonoTests.System.Net.Http
+{
+	[Preserve (AllMembers = true)]
+	public static class NetworkResources
+	{
+		public static readonly string MicrosoftUrl = "https://www.microsoft.com";
```

Comment:
IIRC we moved from microsoft.com to example.com because microsoft.com was giving us trouble earlier this year. Might want to check the history for the example.com urls we had before this PR to see why.

Reply:
Seems to work now, but we can change the name of the var to PublicUrl and then change the real url. Tests seem to pass. I wonder what was changed. Changing the var name seems to be a good idea so that the name does not lie.